angular/PdfViewer/Text-Markup-Annotation.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Text markup annotation of PDF viewer component.
-description: Text markup annotation of PDF viewer component.
+title: Text markup annotation in Angular PDF viewer component | Syncfusion
```

Comment:
Component, C should be in caps also in description

Reply:
Updated